Jenkinsfile

```diff
@@ -42,42 +44,44 @@ pipeline {
         }
       }
       steps {
-        sh """
-          docker build \
-            --build-arg DISABLE_SEARCH_ENGINE \
-            --build-arg GET_CONTENT \
-            --build-arg GATSBY_CONFIG_SITE_METADATA__URL \
-            --build-arg GATSBY_CONFIG_SITE_METADATA__SITE_URL \
-            -t ${imageName()}:${imageTag()} .
-        """
+        sh 'yarn build'
       }
     }
 
-    stage('Docker Push Test') {
-      when {
-        environment name: 'JENKINS_URL', value: 'https://jenkins.gavinmogan.com/'
-      }
-      environment {
-        DOCKER = credentials("dockerhub-halkeye")
-      }
-      steps {
-        sh """
-          docker login --username=\"$DOCKER_USR\" --password=\"$DOCKER_PSW\"
-          docker push ${imageName()}:${imageTag()}
-        """
+		stage('Check build') {
+			steps {
+				sh 'test -e public/index.html || exit 1'
       }
     }
 
+		stage('Lint and Test') {
+			steps {
+				sh 'yarn lint'
+        sh 'yarn test'
+      }
+    }
 
-    stage('Docker Push Production') {
+    stage('Deploy to azure') {
       when {
         environment name: 'JENKINS_URL', value: 'https://trusted.ci.jenkins.io:1443/'
       }
+      environment {
+        STORAGEACCOUNTKEY = credentials('PLUGINSHTML_STORAGEACCOUNTKEY')
+      }
       steps {
-        script {
-          infra.withDockerCredentials {
-            sh "docker push ${imageName()}:${imageTag()}"
-          }
+        /* -> https://github.com/Azure/blobxfer
+           Require credential 'JAVADOC_STORAGEACCOUNTKEY' set to the storage account key */
```

Comment:
Credential name needs updating or comment removed

Reply:
Ah sorted already

Reply:
.. i was very confused, I remember specifically removing that line and didn't think i committed it.